schema/tables/browser_plugins.yml

```
@@ -10,3 +10,4 @@ examples: >-
   SELECT bp.name, bp.identifier, bp.version FROM browser_plugins bp JOIN users u on bp.uid = u.uid ;
 
   ```
+notes: Querying this table requires joining against the `users` table.
```

```suggestion
notes: Querying this table requires joining against the `users` table. [Learn more](https://fleetdm.com/guides/osquery-consider-joining-against-the-users-table)
```

I think we want to a link to Zach's article here and cut the `requires_user_context`. We can also clean up the code in the build script that uses `requires_user_context`.

Why? See reasoning in GitHub comment [here](https://github.com/fleetdm/fleet/issues/16784#issuecomment-2035286424).

Does the website support this? If not, I think let's come back to this in a follow up story.